src/js/themes/hpe.js

```diff
@@ -285,10 +286,17 @@ export const hpe = deepFreeze({
         primary &&
         colorValue &&
         `:hover {
+        color: ${
```

Comment:
This is starting to feel like too much functionality from grommet is leaking into the theme. Maybe we should revert the prior change and re-think how we want to approach this, probably by enhancing the grommet side more.

Reply:
That's what I was thinking maybe we should revert or take out the `colored` button I can make a PR on the grommet side to handle this. I agree this is heavy but didn't think of anything else.